twine/commands/upload.py

```diff
@@ -65,9 +67,9 @@ def _make_package(
 
     if upload_settings.verbose:
```

Comment:
I think one of the benefits of using `logging` is that we don't need to check the setting each time; we can just log at the appropriate level, and let the logging configuration handle the filtering.

```suggestion
```

Reply:
Exactly right, @bhrutledge. Good catch